test/denormalizer/ReduxApiStateDenormalizer.spec.js

```diff
@@ -321,6 +321,20 @@ describe('ReduxApiStateDenormalizer', () => {
       assert.isObject(cachedDenormalizedData[STATUS]);
       assert.isObject(cachedDenormalizedData['type2.test'][STATUS]);
     });
+    it('gets object from cache for unexisting item ', () => {
```

Comment:
This test is based on https://github.com/shoutem/redux-io/blob/develop/test/denormalizer/ReduxApiStateDenormalizer.spec.js#L308 but with `none` value?